src/scripts/lint/.eslintrc.library.js

```diff
@@ -1,25 +1,63 @@
 module.exports = {
-  parser: "@typescript-eslint/parser",
+  parser: '@typescript-eslint/parser',
   extends: [
-    // Uses the recommended rules from @typescript-eslint
-    "plugin:@typescript-eslint/recommended",
-    // Disables ESLint rules that would conflict with prettier
-    "prettier/@typescript-eslint",
-    // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors.
-    // Make sure this is always the last configuration in the extends array.
-    "plugin:prettier/recommended",
+    'eslint-config-airbnb',
```

Comment:
Move this to a different PR and mark it as a feat that updates the listing rules for libraries.